Readme.md

```diff
@@ -3,23 +3,37 @@
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E1300)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
 
-* [Form1.cs](./CS/LineOfBestFit/Form1.cs) (VB: [Form1.vb](./VB/LineOfBestFit/Form1.vb))
-* [Program.cs](./CS/LineOfBestFit/Program.cs) (VB: [Program.vb](./VB/LineOfBestFit/Program.vb))
-<!-- default file list end -->
-# Obsolete - How to implement the best fit function, and apply it to a chart's series
+# How to Implement the Best Fit Function, and Apply it to a Chart's Series
+
+This example shows how to perform a custom regression analysis against scatter series points generated from a data table and plot the line of best fit. 
```

Comment:
plot the line of best fit? phrasing.

Reply:
Also, can you tie the image below with this first sentence? Perhaps you could mention, "...as illustarted on the image below."

Reply:
```suggestion
This example shows how to perform a custom regression analysis against scatter series points generated from a data table and display the Line of Best Fit as illustrated on the image below.
```
*Line of Best Fit* is a term here. https://www.investopedia.com/terms/l/line-of-best-fit.asp

Reply:
This example shows how to perform a custom regression analysis against scatter series points generated from a data table and how to display the Line of Best Fit as illustrated on the image below.